src/deploymentmanager.ts

```diff
@@ -550,36 +544,31 @@ export class DeploymentManager implements IDeploymentManager {
         data.push(`PCS_ARM_ENDPOINT_URL="${this._environment.resourceManagerEndpointUrl}"`);
         data.push(`PCS_AAD_ENDPOINT_URL="${this._environment.activeDirectoryEndpointUrl}"`);
 
-        this.setEnvironmentVariables(data);
+        const cmd = this.generateEnviornmentCommand(data);
+        this.saveEnvironmentVariables(cmd, answers.solutionName);
+        cp.exec(cmd);
+    }
 
-        console.log('Please save the following environment variables to /scripts/local/.env file: \n\ %s', `${chalk.cyan(data.join('\n'))}`);
+    private generateEnviornmentCommand(data: string[]): string {
+        let cmd = '';
+        const osCmdMap = {
+            Darwin: 'launchctl setenv ',
+            Linux: 'export ',
+            Windows_NT: 'SETX ',
+        };
+
+        cmd = data.map((envvar) => osCmdMap[os.type()] + envvar.replace('=', ' ')).join('\n');
+        return cmd;
     }
 
-    private setEnvironmentVariables(data: string[]) {
-        data.forEach((envvar) => {
-            let cmd = '';
-            switch ( os.type() ) {
-                case 'Windows_NT': {
-                    envvar = envvar.replace('=', ' ');
-                    cmd = 'SETX ' + envvar;
-                    break;
-                }
-                case 'Darwin': {
-                    envvar = envvar.replace('=', ' ');
-                    cmd = 'launchctl setenv ' + envvar;
-                    break;
-                }
-                case 'Linux': {
-                    cmd = 'echo ' + envvar + ' >> /etc/environment';
-                    break;
-                }
-                default: { 
-                    console.log('The environment could not be set. unable to determine OS.');
-                    break; 
-                 }
-            }
-            cp.exec(cmd);
-        });
+    private saveEnvironmentVariables(cmd: string, solutionName: string) {
+        const pcsTmpDir: string = `${os.homedir()}${path.sep}.pcs${path.sep}`;
+        let envFilePath: string = `${pcsTmpDir}${solutionName}.env`;
+        if (fs.existsSync(envFilePath)) {
+            envFilePath = `${pcsTmpDir}${solutionName}-${Date.now()}.env`;
+        }
+        fs.writeFileSync(envFilePath, cmd);
+        console.log(`Environment variables are saved into file: '${envFilePath}'`);
```

Comment:
Nit: We should add "You can use these environment variables for future deployments".